lib/install/management/delete.go

```diff
@@ -66,7 +100,7 @@ func (d *Dispatcher) DeleteVCHInstances(conf *metadata.VirtualContainerHostConfi
 		if name == conf.Name {
 			continue
 		}
-		if _, err = d.deleteVM(child); err != nil {
+		if _, err = d.deleteVM(child, d.force); err != nil {
```

Comment:
My concern is that container VM will be in different datastore with appliance VM. Right now, I could assume they are all in the image datastore, but in the future, this still need to be changed.